### PR TITLE
exercises are done

### PR DESCRIPTION
**Pull request by msenaavci**
I have tried to use this code but it is not working. I couldn't add li element in the HTML file. what is the problem?
html.innerHtml+= 'li'

And I finished the exercises.

### CODE REVIEW

File: Week1/exercise/w1/index.js [hunk 1]
```diff
@@ -15,6 +31,19 @@ console.log('header: ', header);
...
+const btnToDo = document.querySelector('#btn-addTodo');
+console.log(btnToDo);
+
+function newToDo () {
+    const toDo = document.querySelector('#todoList');
+    const toDoInput = document.querySelector('#todoInput');
+    console.log(toDoInput.value);
+    var li = document.createElement('li');
+    li.innerHTML = toDoInput.value;   
+    document.body.appendChild(li);
```
Comment (ldealmei) [inline Week1/exercise/w1/index.js:44]: You need to add the \<li\> element to the \<ul\> not the body

**Review by ldealmei — CHANGES_REQUESTED** [review]

File: Week1/exercise/w1/index.js (hunk 1 above)
Comment (msenaavci) [reply Week1/exercise/w1/index.js:44]: document.querySelector('#btn-addTodo');
const btnToDo = document.querySelector('#btn-addTodo');
console.log(btnToDo);

function newToDo () {
    const toDo = document.querySelector('#todoList');
    const toDoInput = document.querySelector('#todoInput');
    console.log(toDoInput.value);
    var li = document.createElement('li');
    li.innerHTML = toDoInput.value; 
    ul.appendChild(li);
}
btnToDo.addEventListener('click', newToDo);

Comment (segersrobbertKlarrio) [reply Week1/exercise/w1/index.js:44]: Don't put code in comments, update your pull request by pushing new commits

Comment (segersrobbertKlarrio) [reply Week1/exercise/w1/index.js:44]: you are doing `ul.appendChild(li);` but the variable `ul` is not defined. First declare that variable, assign the list element as the value of that variable, then you can append a child element to that list

**Review by segersrobbertKlarrio — CHANGES_REQUESTED** [review]
you are doing ul.appendChild(li); but the variable ul is not defined. First declare that variable, assign the list element as the value of that variable, then you can append a child element to that list

File: Week1/exercise/w1/index.js (hunk 1 above)
Comment (msenaavci) [reply Week1/exercise/w1/index.js:44]:
```js 
document.querySelector('#btn-addTodo');
const btnToDo = document.querySelector('#btn-addTodo');
console.log(btnToDo);

function newToDo () {
const toDo = document.querySelector('#todoList');
const toDoInput = document.querySelector('#todoInput');
console.log(toDoInput.value);
var li = document.createElement('li');
li.innerHTML = toDoInput.value;
const ul = document.querySelector('ul');
ul.appendChild(li);
}
btnToDo.addEventListener('click', newToDo);
```

Comment (msenaavci) [reply Week1/exercise/w1/index.js]: I thought ul is defined in the HTML file, so I shouldn't declare it again. But now, I understand. Thank you for warning me.

Comment (segersrobbertKlarrio) [reply Week1/exercise/w1/index.js:44]: Please do NOT put code in comments, update your pull request instead by making new commits

Comment (msenaavci) [reply Week1/exercise/w1/index.js:44]: I updated my pull request but it sent my new file to other my pull request called 'homework week-1 is done'. Is it okay now?

**Pull request closed without merging**